Packs/PrismaCloud/Integrations/RedLock/RedLock.py

```diff
@@ -407,7 +411,7 @@ def get_rql_response():
         'Type': entryTypes['note'],
         'ContentsFormat': formats['json'],
         'Contents': response,
-        'EntryContext': {'Redlock':response},
+        'EntryContext': {'Redlock.Rql(val.query === obj.query)': response},
```

Comment:
```suggestion
        'EntryContext': {'Redlock.RQL(val.query === obj.query)': response},
```

Reply:
I have found that the output of this command can actually be quite large (~5000 lines in sandbox env).  Does it make sense to not save the output to context by default in that case? I think that has been done on other integrations in the past.

Reply:
else I will update with your suggestion above, thanks!

Reply:
This hasn't gone through a design with our security team?
@ShahafBenYakir @Arsenikr FYI

Reply:
As I can see in the docs page (https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-rql-reference/rql-reference/operators.html) we can add a limit to the RQL query, so let's add an argument to the command named `limit` and pass it to the query (in the correct syntax)

Reply:
Ok sweet, I thought that must exist somewhere.  Added it now as a command parameter.  Also updated the docs and output for RQL command.  Working on getting an example for the remediate command to do the same now.